app/assets/stylesheets/addtocalendar.scss

```diff
@@ -107,3 +107,5 @@
 .atc-style-blue .atcb-item:hover {
     background: $text-blue;
 }
+
+.atcb-item-link { font-family: $body-font !important; }
```

Comment:
Can you format this like the above e.g. in 3 lines? Is this `important` required: no worries if so and can you remove if not? Thanks!